apps/www/components/Card.tsx

```diff
@@ -56,21 +56,21 @@ const Card = (props: Props) => {
       target="_blank"
       className="relative flex transform flex-col overflow-hidden rounded-md shadow-lg transition hover:-translate-y-1 hover:shadow-2xl"
     >
-      <div className="dark:bg-dark-800 flex-shrink-0">
-        <img className="h-64 w-full object-cover" src={imgUrl} alt={title} />
+      <div className="flex-shrink-0">
+        <img className="h-64 w-full text-light object-cover" src={imgUrl} alt={title} />
```

Comment:
`text-foreground-light`

Reply:
thought we were following the 'foreground-x' convention ?

Reply:
You're right, it's probably better to keep it more explicit to reduce confusion. I'll update it 👍

Reply:
On the other hand, doing a "Find all" in vscode reveals that there are already several occurrences where `text-light` and `text-lighter` have been used across docs/studio/www. This might be a symptom that actually omitting the `foreground` part when used on text might be good UX. Or developers tend to prefer writing that shorthand for velocity. Wdyt? @MildTomato

Reply:
I reckon the thing that caused the issue was just the default.. so `text`, so maybe we just deal with that and just use `text-default` so its easy to find, and then we still get to use shorthand everywhere else.

I also picked up on others using the shorthand 😂 might as well lean into it.

Reply:
I do personally like shorthand :D

Reply:
Ah, for just `text` I leaned on the `text-foreground` shorthand 😅 It felt more clear to what it was referring to.

Reply:
shall we lean hard into one or the other?

mixing -foreground with shorthand can make it more confusing.

Reply:
`text-default` doesn't seem to work though.

Reply:
Nevermind, I keep forgetting it's `text-DEFAULT`.

Reply:
Though I think I'd rather prefer to keep `text-foreground-light` etc not shorthand, but have `text-foreground` instead of `text-default`. For the sake of consistency, clarity and legibility.

Reply:
> text-default doesn't seem to work though. (nor does text-DEFAULT)

we could make it work. probably an overshight.

let's go with what you prefer though.